microservices/front-end/vite.config.ts

```diff
@@ -1,13 +1,17 @@
-import { defineConfig } from "vite";
+import { defineConfig, loadEnv } from "vite";
 import react from "@vitejs/plugin-react";
 import Svgr from "vite-plugin-svgr";
 
 // https://vitejs.dev/config/
-export default defineConfig({
-  plugins: [react(), Svgr()],
-  server: {
-    proxy: {
-      "/api": process.env.AUTH_SERVICE_URL ?? "http://localhost:8080",
+export default defineConfig(({ mode }) => {
+  const env = loadEnv(mode, process.cwd())
+  console.log('env '+ JSON.stringify(env));
```

Comment:
I don't think this is needed

Reply:
I need it to set auth server url for deployment

Reply:
the comment is just for the console log

Reply:
Fixed in [58b5c38](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g37/pull/39/commits/58b5c38fe55b22e7605daeab6768eb7cea965145)